### ISSUE TITLE
error: pathspec 'new' did not match any file(s) known to git during installation

### ISSUE THREAD

**Issue by siiron** [intuit/design-systems-cli#108]
**Describe the bug**

Install script fails. 

**To Reproduce**

Clean install on Windows following [documentation](https://intuit.github.io/design-systems-cli/#/getting-started).

**Expected behavior**

To finish install without errors.

**Desktop:**

- Windows 10 Enterprise, 10.0.18363, 
- Node v11.4.0
- Yarn v1.17.3

**Additional context**

Here is the entire output log:

```
PS C:\Users\username\Dev\Projects> npm init @design-systems
npx: installed 1 in 3.529s
[create....] » 🤞  pending      Cloning template...
? What's the system name? ds
? What's your name? username
? What's your email? username@company.com
? Starting version? You probably don't want to change this 0.0.0
? Repository url or owner/repo: https://github.com/..../ds
[create....] » 💾  info         Initializing git repo
[create....] » ⏳  awaiting     Running yarn. This may take a bit...
✓ Installing dependencies 1 ms
error: pathspec 'new' did not match any file(s) known to git
error: pathspec 'design' did not match any file(s) known to git
error: pathspec 'system'' did not match any file(s) known to git
(node:480) UnhandledPromiseRejectionWarning: Error: Command failed: cd ds && git add . && git commit --no-verify -m 'Create new design system'
error: pathspec 'new' did not match any file(s) known to git
error: pathspec 'design' did not match any file(s) known to git
error: pathspec 'system'' did not match any file(s) known to git

    at checkExecSyncError (child_process.js:639:11)
    at Object.execSync (child_process.js:676:15)
    at Object.e.default (C:\Users\username\AppData\Roaming\npm-cache\_npx\25356\node_modules\@design-systems\create\dist\bundle.js:194:425)
    at processTicksAndRejections (internal/process/task_queues.js:86:5)
(node:480) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:480) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.6.1`](https://github.com/intuit/design-systems-cli/releases/tag/v2.6.1) :rocket:

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.6.1`](https://github.com/intuit/design-systems-cli/releases/tag/v2.6.1) :rocket:
